avaje-config/src/main/java/io/avaje/config/Configuration.java

```diff
@@ -482,6 +482,15 @@ default boolean enabled(String key, boolean enabledDefault) {
    */
   void loadIntoSystemProperties();
 
+  /**
+   * Trigger a refresh on all {@link ConfigurationSource}.
+   * <p>
+   * Generally configuration sources will schedule a periodic refresh of their
+   * configuration but there are cases like Lambda where it can be useful to
+   * trigger a refresh explicitly (e.g. on Lambda invocation).
+   */
+  void refresh();
```

Comment:
perhaps a more descriptive name? At first glance, this implies it can be used to reload all the configuration in general.

Reply:
Renamed to `reloadSources()`